data/trie/trieStorageManager.go

```diff
@@ -511,17 +513,17 @@ func (tsm *trieStorageManager) removeSnapshot() {
 
 func disconnectSnapshot(db data.DBWriteCacher) error {
 	err := db.Close()
-	if err != nil {
+	/*if err != nil {
```

Comment:
Remove commented code. You could use `return db.Close()`

Reply:
Done!